moveit_demo_nodes/run_move_group/launch/run_move_group_interface.launch.py

```diff
@@ -3,53 +3,18 @@
 from launch import LaunchDescription
 from launch_ros.actions import Node
 from ament_index_python.packages import get_package_share_directory
-
-
-def load_file(package_name, file_path):
-    package_path = get_package_share_directory(package_name)
-    absolute_file_path = os.path.join(package_path, file_path)
-
-    try:
-        with open(absolute_file_path, "r") as file:
-            return file.read()
-    except EnvironmentError:  # parent of IOError, OSError *and* WindowsError where available
-        return None
-
-
-def load_yaml(package_name, file_path):
-    package_path = get_package_share_directory(package_name)
-    absolute_file_path = os.path.join(package_path, file_path)
-
-    try:
-        with open(absolute_file_path, "r") as file:
-            return yaml.safe_load(file)
-    except EnvironmentError:  # parent of IOError, OSError *and* WindowsError where available
-        return None
+from moveit_configs_utils.moveit_configs_builder import MoveItConfigsBuilder
 
 
 def generate_launch_description():
-    # planning_context
-    robot_description_config = load_file(
-        "moveit_resources_panda_description", "urdf/panda.urdf"
+    moveit_config = (
+        MoveItConfigsBuilder("moveit_resources_panda")
+        .robot_description(file_name="panda.urdf.xacro")
+        .robot_description_semantic()
+        .robot_description_kinematics()
+        .joint_limits()
+        .moveit_configs()
```

Comment:
Calling like this seems a bit strange, but I like that it allows you to decide which functions to call, and pass parameters for each.

We should make it clear that you need `.moveit_configs()` last no matter what

Reply:
I don't know how to do it in python but the approach normally used in cpp is to make so the builder object has a conversion operator to the final type so it can be converted.  In python the types are dynamic so it means you probably need a function like this that'll return the moveit configs object.  Maybe there was some sort of to_dict() or something you could overload so the moveit_config object would just be converted into a dictionary when you use it like one?